src/Entities/ArmorStand.cpp

```diff
@@ -0,0 +1,461 @@
+
+#include "Globals.h"  // NOTE: MSVC stupidness requires this to be the same across all modules
+
+#include "ArmorStand.h"
+#include "Player.h"
+#include "../ClientHandle.h"
+#include "../Chunk.h"
+#include "../World.h"
+
+
+
+
+
+cArmorStand::cArmorStand(Vector3d a_Pos, double a_Yaw, short a_Size):
+	cEntity(etArmorStand, a_Pos.x, a_Pos.y, a_Pos.z, 0.25*a_Size, 0.9875*a_Size),  // a_Size = 0 : Marker ;  a_Size = 1 : Small ;  a_Size = 0 : Normal
```

Comment:
What exactly is marker and why does it is controlled by `m_Size`? Would a small marker not be a valid use case?

Reply:
According to:
https://wiki.vg/Entity_metadata#Objects
https://minecraft.gamepedia.com/Armor_Stand

Marker state of an armour stand highly depend of size. But I think that I will prefere two arguments bool a_Small and bool a_Marker. That's smaller in term and ram and it will be more clear to understand

Reply:
Thanks for the wiki link but I'm not sure if "size" there is only referring to the collision volume. Based off the fact that they're both booleans, it sounds like it should be possible to have normal and small markers though.

Reply:
Then I will separate m_Size to m_Marker and m_Small.
But isn't enough to have setHeight and setWidth available separatly ?
Because the wiki define size according to the type of armor stand is it.
At the other hand I agree that is not a clear definition of what size should be; it's not defined bidirectionally

Extract:
> Marker: 1 or 0 (true/false) - if true, ArmorStand's size will be set to 0, have a tiny hitbox and disable interactions with it. May not exist.

> Name | bounding box x and z | bounding box y
> ArmorStand | normal: 0.5 marker: 0.0 small: 0.25 | normal: 1.975 marker: 0.0 small: 0.9875

Reply:
>or should I simply replace m_Size by m_Marker and m_Small ?

I think so, although I would call them `m_IsMarker` and `m_IsSmall`.

Reply:
Done